core/services/flux_monitor.go

```diff
@@ -426,9 +432,7 @@ func (p *PollingDeviationChecker) consume(ctx context.Context, roundSubscription
 			err := p.respondToNewRound(log)
 			logger.ErrorIf(err, "checker unable to respond to new round")
 		case <-time.After(p.delay):
-			ok, err := p.poll(p.threshold)
-			logger.ErrorIf(err, "checker unable to poll")
-			jobRunTriggered = ok
+			jobRunTriggered = p.pollIFRoundOpen(client)
 		case <-idleThresholdTimer.C:
 			ok, err := p.poll(0)
```

Comment:
Should we use `pollIfRoundOpen` here too?

Reply:
@spooktheducks This the "heartbeat" and so the FM should try to start a new round whenever this timer expires. There is an issue though, where the round could still be closed and this would error. I have a new story pending to make sure the PA's timeout is <= idleThreshold, which I think is the more appropriate way to deal with this edge case.

https://www.pivotaltracker.com/story/show/171468269